studio/src/editor-panel.js

```diff
@@ -212,6 +212,12 @@ export default class EditorPanel extends LitElement {
         this.hasChanges = true;
     }
 
+    #handleLocReady() {
```

Comment:
had similar change in my ticket for a checkbox and we could result the usual handle method 
but this way is also ok